Add SPM support

mchoo7 opened the pull request:
I used [OrbotKit.podspec](https://github.com/guardianproject/orbotkit/blob/main/OrbotKit.podspec) to create `Package.swift` file. 

iOS version has been bumped to iOS 15 since iOS 14 and older are now [obsolete](https://en.wikipedia.org/wiki/IOS_14).

mchoo7 committed:
Add SPM support

mchoo7 committed:
Move OrbitKit sources to Sources folder

mchoo7 committed:
Bump version from iOS 11 to iOS 15

mchoo7 committed:
Remove swift language version reference

mchoo7 committed:
Bump swift-tool-version to 5.5

mchoo7 committed:
Update README.md for Swift Package Manager

mchoo7 committed:
Add .swiftpm for Xcode

tladesignz commented:
Hm. Seems, you're a little ahead of the curve.
This is very interesting, but I can't accept it with an Xcode beta build.

Also, even though old iOS versions are deprecated, there's devices out there, which use it. Some apps, which support Orbot, might also want to run on old iOS versions. (Even though Orbot itself doesn't run there.)

If there's a technical reason, why this lib cannot support old iOS versions, I'm happy to push forward, but otherwise, this lib's support stays where it is.

mchoo7 committed:
Revert iOS version to iOS 11

mchoo7 committed:
Restore MARKETING_VERSION in project.pbxproj

mchoo7 commented:
> Hm. Seems, you're a little ahead of the curve. This is very interesting, but I can't accept it with an Xcode beta build.
> 
> Also, even though old iOS versions are deprecated, there's devices out there, which use it. Some apps, which support Orbot, might also want to run on old iOS versions. (Even though Orbot itself doesn't run there.)
> 
> If there's a technical reason, why this lib cannot support old iOS versions, I'm happy to push forward, but otherwise, this lib's support stays where it is.

I reverted iOS version, Xcode version, and updated Xcode settings.

tladesignz commented:
> I reverted iOS version, Xcode version, and updated Xcode settings.

Great! Thank you!
